samples/21_sound_and_effects.cr

```diff
@@ -0,0 +1,62 @@
+require "../src/sdl"
+require "../src/image"
+require "../src/mix"
+
+SDL.init(SDL::Init::VIDEO | SDL::Init::AUDIO); at_exit { SDL.quit }
+SDL::MIX.init(SDL::MIX::Init::MP3); at_exit { SDL::MIX.quit }
```

Comment:
I can't init with MP3 on Ubuntu with the system packages, because SDL_mixer is compiled without MP3 support. Using `FLAC` instead works. I'm surprised there is no `WAV` type.